drivers.c

```diff
@@ -39,6 +40,9 @@
 #define USER_META_KEY "user-meta"
 #define DEV_META_KEY "device-meta"
 
+// modprobe + remove option + module name + module options
+#define DRIVER_MGMT_COMMAND "/sbin/modprobe %s %s %s"
```

Comment:
is it possible to call kmod directly? In that way its easier to understand why something isn't working if you havent enabled KMOD in global.config....

Reply:
Ok, got it, but my initial idea was to replace the old modprobe invocation in https://github.com/pantavisor/pantavisor/blob/feature/mdev/drivers.c#L164 with this in the future. I don't really like to have 2 places where we use a very similar command. So the idea was to use `pv_drivers_load_single` and `pv_drivers_unload_single`.